tests/test_usuarios.py

```
@@ -0,0 +1,251 @@
+import pytest
```

Ruff (0.11.9)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)